model-optimizer/extensions/ops/BatchNormInferenceMultipleOutputs.py

```diff
@@ -0,0 +1,42 @@
+"""
+ Copyright (C) 2018-2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from mo.graph.graph import Graph
+from mo.ops.op import Op
+import numpy as np
+
+
+class BatchNormInferenceMO(Op):
+    """
+    BatchNormInference with Multiple outputs
```

Comment:
Please, extend the description of this operation. I think it make sense to add that this operation is used to extract TensorFlow FusedBatchNorm operation.

Reply:
done